widget/button.go

```diff
@@ -182,6 +191,7 @@ const (
 	DefaultButton ButtonStyle = iota
 	// PrimaryButton that should be more prominent to the user
 	PrimaryButton
+	CancelButton
```

Comment:
It's not clear what the "CancelButton" style is - and no doc to explain

Reply:
This is another hack to implement the functionality of default buttons associated with the enter key and escape key. Normaly, for dialogs there is a default "OK" or "Submit" key that is activated by pressing Enter/Return, and a default "Cancel" or "Abort" key that is activated by pressing "Escape". This is the way I found to distinguish the types. Perhaps there is a better name than "CancelButton", or we could have another property of the buttons. In Delphi there is both Cancel and Default boolean properties that makes the button respond to Enter/Escape. But I think the new style is a better solution.

Reply:
Is this specific to forms as well?